### PR TITLE
Gemstones: Containers: Podman: Fix typos and admonitions

### PR DESCRIPTION
**Pull request by ChrSteinert**
As discussed.

#### Author checklist (Completed by original Author)
- [x] Good fit for the Rocky Linux project? Title and Author Metatags inserted ?
- [x] If applicable, steps and instructions have been tested to work
- [x] Initial self-review to fix basic typos and grammar completed

#### Rocky Documentation checklist (Completed by Rocky team) 
- [x] 1st Pass (Document is good fit for project and author checklist completed)
- [x] 2nd Pass (Technical Review - check for technical correctness) 
- [x] 3rd Pass (Detailed Editorial Review and Peer Review)
- [x] Final approval (Final Review)

### CODE REVIEW

File: docs/gemstones/containers/podman.md [hunk 1]
```diff
@@ -75,8 +75,9 @@ To now run your service run:
 systemctl --user start nextcloud.service
 ```
 
-> [!NOTE]
-> If you created a file in one of the directories for rootful services, omit the `--user` flag.
+!!! note
```
Comment (SergeCroise) [inline docs/gemstones/containers/podman.md:78]:
```suggestion
!!! note "Note"
```

File: docs/gemstones/containers/podman.md [hunk 2]
```diff
@@ -98,8 +99,9 @@ Other file types are supported: pod, volume, network, image and kube. [Pods](htt
 
 Podman additionally provides the `generate systemd` subcommand. It can be used to generate `systemd` service files. 
 
-> [!IMPORTANT]
-> `generate systemd` is now deprecated and will not receive further features. Usage of Quadlet is recommendet.
+!!! warning
```
Comment (SergeCroise) [inline docs/gemstones/containers/podman.md:102]:
```suggestion
!!! warning "Warning"
```

**Comment by ChrSteinert** [comment]
I will apply those changes to #2108 as well.

File: docs/gemstones/containers/podman.md (hunk 2 above)
Comment (SergeCroise) [reply docs/gemstones/containers/podman.md:102]: I prefer this syntax to make the translation work easier
(feel free to modify or revert my changes)

Comment (ChrSteinert) [reply docs/gemstones/containers/podman.md]: I trust your experience on that one… And I do not have any opinion on that :D

**Comment by ChrSteinert** [comment]
I added explicit headers to all the notes. Also in #2108

**Comment by sspencerwire** [comment]
@ChrSteinert I see that this suggestion came from @SergeCroise, however there was a debate about this within the localization channel of Mattermost, and it was decided then that it was unneeded. If you want to bring that up again, @SergeCroise (in Localization) and see if opinions have changed, I'm fine with that. I think the consensus was that it added another layer of complexity to writing the original English source and wasn't really needed. If that opinion has changed, I'm willing to revisit.  Thank you both!

**Review by sspencerwire — APPROVED** [review]

**Pull request merged**